Dev guide

indiegamehunt opened the pull request.

indiegamehunt committed:
Added dev guide.

indiegamehunt committed:
Fixed spacing I hadn't noticed before

niksudan commented:
'Initial commit' isn't necessary, it's just what people use as their first commits when creating a repository. Since you're branching it doesn't matter what you call it, as technically it isn't the initial commit of the whole project. Remember - when it's merged your commits become part of master, so it seems out of place to have two 'Initial commits'.

hworld commented on dev-guide/index.md:
```diff
@@ -0,0 +1,331 @@
+
+Speaking of links, linking your social media accounts with Game Jolt is more important than you might think. Don't put your Twitter and Facebook links in the body of your profile; [link them](http://gamejolt.com/dashboard/linked-accounts/) directly. They will appear big and bold on the side of your profile, which makes it easier for fans, press, staff, etc. to contact you.
+
+* * *
```
Can we change these to three hyphens? `---`

indiegamehunt replied:
Sure!

hworld commented on dev-guide/index.md:
```diff
@@ -0,0 +1,331 @@
+
+**Title:** Surely your game has a name. Put it here.
+
+- *A note on mature language:* If your game's title contains the "F-word" or a swear word of that magnitude, please replace one or more of its letters with *. This will keep browsing friendly for people who do not wish to view mature content. This only applies to titles and thumbnails because they show up in lists and search results; once someone visits a game page, there is a maturity rating on  ...
```
Cancel out the \* by either putting it in backticks, or by putting a backslash before. It currently is trying to make it italic.

indiegamehunt replied:
Actually, italics was my intention... should I change it?

hworld commented on dev-guide/index.md:
```diff
@@ -0,0 +1,331 @@
+
+**Development Status:** You must indicate your game's development status. You can choose one of three options:
+
+- *Finished:* Your game is complete. You're uploading a build, and that build is stable. You might upload fixes or other small updates, but you don't expect to make any major changes. 
```
Let's use `_` for italics and `**` for bold.

indiegamehunt replied:
Sounds good. Edited.

indiegamehunt committed:
Changed *** to ---

indiegamehunt committed:
Fixed italic and bold formatting

Changed italics to bold in places where it hadn't converted from the rtf
properly (I hadn't caught this before). Changed intentional use of
italics from * to _.

indiegamehunt committed:
Made "tip" bold

indiegamehunt committed:
Added italics

indiegamehunt committed:
Changed *

MaskofThoro commented on dev-guide/index.md:
```diff
@@ -0,0 +1,331 @@
+
+If you already have a gamer account on Game Jolt, it's a cinch to become a developer. Just hover your mouse over "Dashboard" (on the navigation bar at the top of the page) until a new menu appears, then click/tap "Become a Developer".
+
+### Your Avatar
```
we should move this out of the "dev" guide and into a general guide since avatars are used by both devs and gamers.

MaskofThoro commented on dev-guide/index.md:
```diff
@@ -0,0 +1,331 @@
+
+Game Jolt uses Gravatar for avatars. Sign up for an account at [gravatar.com](http://gravatar.com/) using the same email address you use for Game Jolt and upload your avatar image. It will automatically begin appearing throughout the site. If you already use Gravatar, you've probably already seen your avatar on the site. If you want it to be something else, upload a new image to Gravatar and asso ...
+
+### Your Profile
```
move to a general guide since profile is used by both devs and gamers.

indiegamehunt commented:
Obsolete
